### ISSUE TITLE
insert data into database table directly

### ISSUE THREAD

**Issue by shokufeh-coder** [benkeen/generatedata#534]

**Comment by benkeen** [comment]
Nope, sorry @shokufeh-coder. It'll only ever generate the SQL - I don't have any plans to actually connect to a DB and run the generated code; that'll have to be done separately.
